Classes/common/CDTFetchChanges.h

```diff
@@ -0,0 +1,175 @@
+//
+//  CDTFetchChanges.h
+//  CloudantSync
+//
+//  Created by Michael Rhodes on 31/03/2015.
+//  Copyright (c) 2015 IBM Cloudant. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+//  except in compliance with the License. You may obtain a copy of the License at
+//    http://www.apache.org/licenses/LICENSE-2.0
+//  Unless required by applicable law or agreed to in writing, software distributed under the
+//  License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+//  either express or implied. See the License for the specific language governing permissions
+//  and limitations under the License.
+
+#import <Foundation/Foundation.h>
+
+@class CDTDatastore;
+@class CDTDocumentRevision;
+
+/**
+ CDTFetchChanges objects report changes to a database, that is, documents which have been
+ created, updated or deleted. This feature can be used to keep external datastores up to
+ date with changes in a database, for example to keep an external index of documents up
```

Comment:
It's unclear to me what this means... " This feature can be used to keep external datastores up to date with changes in a database". Is it to sync with remote Cloudant databases? 

I guess as an app developer, I'd like to see from the very first paragraph how it would be used in my app. The second sentence progresses quickly to a low level details. But maybe it's not envisioned to be used at a higher level. It's hard for me to say what I want you to write here, so I've no particular suggestion. I guess as an app developer, I'd like a better description of why I'd use this class.

Also, the second sentence could be broken up. "This feature can be used to.... .  For example, CDTQuery use CDTFetchChanges internally in the Query implementation to keep indexes up to date." (I suppose this will actually be implemented later?)